src/main/java/io/axoniq/axonserver/connector/impl/FlowControlledBuffer.java

```diff
@@ -56,6 +60,9 @@ public FlowControlledBuffer(String clientId, int bufferSize, int refillBatch) {
 
     @Override
     public void onNext(T value) {
+        if (value == null) {
```

Comment:
There is a shorthand for this: `Objects.requireNonNull((...)`. Not sure if it's available in Java 8, though.